### DISCUSSION TITLE
这个是什么情况啊，网上的方法都试遍了还是不可以（What's the situation? I've tried all the methods online, but it still doesn't work）

### DISCUSSION THREAD

**Discussion by lntomF** [hexojs/hexo#5570] [Question / Answer] [answered]
![屏幕截图 2024-11-03 005814](https://github.com/user-attachments/assets/e7e1fa61-1f24-460f-9573-3f91e26275e9)
我是用的是淘宝最新npm源

**Comment by uiolee** [comment]
![image](https://github.com/user-attachments/assets/e9e29346-5dc2-4f1c-a433-acd5f178ab11)

**Reply by lntomF** [reply]
我已经试过这个命令也是一推报错，为什么在git bash里面使用时会出现报错在我用管理员命令窗口在blog文件下使用相关命令就不会有问题，而且在bash的窗口下我无法使用npm 下载东西

**Reply by lntomF** [reply]
求大佬解答

**Reply by uiolee** [reply]
> 一推报错

post the log

**Reply by uiolee** [reply]
> git bash里面使用时会出现报错
> 在bash的窗口下我无法使用npm 下载东西

This is most likely related to npm or node, which you may not have installed correctly. Try reinstalling. This has nothing to do with hexo

**Reply by lntomF** [reply]
![屏幕截图 2024-11-03 184914](https://github.com/user-attachments/assets/04a5fad5-d4f6-473d-af5e-7828633ca99d)
如果我用bash就会出现这样的情况 node我已正确安装最新版本
![Uploading 屏幕截图 2024-11-03 185048.png…]()

**Reply by uiolee** [reply, marked as answer]
https://www.google.com/search?q=EPERM
